reference/redox.v1.yaml

```diff
@@ -0,0 +1,35 @@
+openapi: 3.0.0
+x-stoplight:
+  id: gpm2ktdetzht7
+info:
+  title: Redox
+  version: '1.0'
+  description: |-
+    This spec is used for generating models for incomming Redox messages. In the future it may also be used for generating an SDK library for pushing results to Redox.
+
+    To enable code generation for a specific model you should reference it in the request body `/endpoint` API endpoint.
+servers:
+  - url: 'https://api.redoxengine.com/endpoint'
+paths:
+  /endpoint:
+    post:
+      summary: Send Data
+      tags: []
+      responses: {}
```

Comment:
Should we include the definition of a response? https://docs.redoxengine.com/basics/data-exchange-with-redox/responding-to-asynchronous-events/#responses-to-send

Reply:
If we use this to generate a client - yes. I'll do this in a follow up task when I get there.